airflow/cli/commands/role_command.py

```diff
@@ -42,3 +45,45 @@ def roles_create(args):
     for role_name in args.role:
         appbuilder.sm.add_role(role_name)
     print(f"Added {len(args.role)} role(s)")
+
+
+@suppress_logs_and_warning
+def roles_export(args):
+    """
+    Exports all the rules from the data base to a file.
+    Note, this function does not export the permissions associated for each role.
+    Strictly, it exports the role names into the passed role json file.
+    """
+    appbuilder = cached_app().appbuilder
+    roles = appbuilder.sm.get_all_roles()
+    exporting_roles = [role.name for role in roles if role.name not in EXISTING_ROLES]
+    filename = os.path.expanduser(args.file)
+    with open(filename, 'w') as file:
+        file.write(json.dumps(exporting_roles, sort_keys=True, indent=4))
```

Comment:
```suggestion
    with open(filename, 'w') as f:
        json.dump(f, exporting_roles, sort_keys=True, indent=4)
```

I feel it may be worthwhile to add some formatting options to the command, since both `sort_keys` and `indent` have performance considerations and a user may reasonably want to not set those if there's a large amount of data. Maybe we should even default to _not_ pretty-format the output.